controller/main_controller.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ *
+ * Collapsible Categories extension for the phpBB Forum Software package.
+ *
+ * @copyright (c) 2015 phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ */
+
+namespace phpbb\collapsiblecategories\controller;
+
+class main_controller implements main_interface
+{
+	/** @var \phpbb\collapsiblecategories\operator\operator */
+	protected $operator;
+
+	/** @var \phpbb\request\request */
+	protected $request;
+
+	/**
+	* Constructor
+	*
+	* @param \phpbb\collapsiblecategories\operator\operator $operator Operator object
+	* @param \phpbb\request\request                         $request  Request object
+	* @access public
+	*/
```

Comment:
Last little thing, the spacing should be consistent for function docblocks, and we are doing 1 space before the asterix
```
/**
 *
 *
``